registry/biocontainers/picard/container.yaml

```diff
@@ -1,12 +1,17 @@
 docker: biocontainers/picard
 latest:
- "v1.139_cv3": "sha256:05a6dd401d47c930191fa8c017b2cf60c099c4da236f134586a2f3610e426229"
+  v2.3.0_cv3: sha256:8624a229b70fc495e29f55f67ec630e30fa43db3f2a5e276e0e0a68551fee0ea
 tags:
- "v1.139_cv3": "sha256:05a6dd401d47c930191fa8c017b2cf60c099c4da236f134586a2f3610e426229"
+  v1.139_cv3: sha256:05a6dd401d47c930191fa8c017b2cf60c099c4da236f134586a2f3610e426229
+  2.3.0: sha256:5e1bc7788a3aa1329e59821e3f89b5bb209372926a67ed1a95f7b9a71350e2be
+  v2.3.0_cv3: sha256:8624a229b70fc495e29f55f67ec630e30fa43db3f2a5e276e0e0a68551fee0ea
+  '1.141': sha256:675128282859c9edf1c8679478c828a48d980a4e73b95abba6a087d315d01cfc
```

Comment:
I'm curious about this. Without the quotes, the yaml parser would transform this to a float ?

Reply:
All the tags should be parsed as strings and the pieces are split into int.

Reply:
So if the quotes are unnecessary, can they be removed ? I find it disturbing that some tags have quotes and some don't

Reply:
The quotes are added by the yaml parser, and we use a specific one that preserves comments. I don't know any way to remove them, but I don't think they hurt. Is there a reason to not want them there other than aesthetic?